kubernetes/bin/update-deployment-travis.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+if [ "$TRAVIS_REPO_SLUG" != "loklak/loklak_server" ]; then
+    echo "Skipping image update for repo $TRAVIS_REPO_SLUG"
+    exit 0
+fi
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
+    echo "Skipping image update for pull request"
+    exit 0
+fi
+
+if [ "$TRAVIS_BRANCH" != "master" ] && [ "$TRAVIS_BRANCH" != "development" ]; then
+    echo "Skipping image update for branch $TRAVIS_BRANCH"
+    exit 0
+fi
+
```

Comment:
checking for conditions having `$TRAVIS_REPO_SLUG`, `$TRAVIS_PULL_REQUEST` and `$TRAVIS_BRANCH` are quite similar.
IMHO handling these conditions for Loklak Server at one place would be better. :)